MovieApp/PresentationLayer/Views/Components/MovieInfoCell/MovieInfoCell.swift

```diff
@@ -0,0 +1,52 @@
+import UIKit
+
+class MovieInfoCell: UICollectionViewCell {
+
+    static let cellIdentifier = "movieInfoCell"
```

Comment:
we usually define this as:
`static let cellIdentifier = String(describing: MovieInfoCell.self)`